internal/voice/stream/connect.go

```diff
@@ -59,7 +59,7 @@ func (strm *Streamer) newChipperConn(ctx context.Context) (Conn, *CloudError) {
 			},
 		}
 
-		otaURL = "https://" + config.Env.Check + suffix
+		otaURL = "http://" + config.Env.Check + suffix
```

Comment:
I don't think we need this anymore.
With the 1.8 escape pod update, escape pod runs fake https so Vector can send at least HTTPS GET request to escape pod.

Reply:
You are right